Composer/cypress/integration/LUPage.spec.ts

```diff
@@ -8,7 +8,7 @@ context('LU Page', () => {
   });
 
   it('can open language understanding page', () => {
-    cy.findByTestId('LeftNav-CommandBarButtonUser Input').click();
+    cy.findByTestId('LeftNav-CommandBarButtonUser Input').click({ force: true });
```

Comment:
Let's make this work without using `force`.

Reply:
I'm not sure we can. Without `force`, the e2e tests were failing because, while Cypress could see the element it wanted to click, it complained that the element was being covered by something, and some investigation showed that that something else was the tooltip's active region. I could see if making Cypress click on the tooltip region works, but that doesn't seem like a natural way to test the behavior we actually want.

Reply:
On further inspection, the e2e tests are still failing even with `force`, so I'll figure out another way to do this.

Reply:
Thanks for looking at this.

Reply:
It looks like those failures were transient. They were definitely failing without `force`, though, so I think we still might need it.

Reply:
We should figure out why we need it. To me, this lowers my confidence in these tests because it could lead to false positives.